GTMAppAuthSwift/Tests/GTMAppAuthFetcherAuthorizationTests.swift

```diff
@@ -358,10 +358,12 @@ class GTMAppAuthFetcherAuthorizationTest: XCTestCase {
       try GTMAppAuthFetcherAuthorization.removeAuthorization(for: testKeychainItemName)
       XCTAssertFalse(keychainHelper.useDataProtectionKeychain)
     } catch {
-      guard let keychainError = error as? KeychainWrapper.Error else {
-        return XCTFail("`error` should be of type `GTMKeychainManager.Error`")
+      guard let keychainError = error as? GTMAppAuthFetcherAuthorization.Error else {
+        return XCTFail("`error` should be of type `GTMAppAuthFetcherAuthorization.Error`")
       }
-      XCTAssertEqual(keychainError, KeychainWrapper.Error.failedToDeletePasswordBecauseItemNotFound)
+      XCTAssertEqual(
```

Comment:
I'm open to adding more. Do you have any in mind?

Reply:
I was thinking of tests for the exception scenarios in `authorizationForItemName:error:` and `saveAuthorization:withItemName:error:`, but maybe this would be more appropriate as part of a subsequent testing pass.  Since this PR will be merged to your branch, I leave it to you.

Reply:
There is an existing test for the case of trying to get an authorization for a missing item name. I added a test that verifies the error thrown when trying to save an item with an empty item name.